docs/source/guide/events.rst

```diff
@@ -212,11 +212,51 @@ Boto3 specific events
 Boto3 emits a set of events that users can register to
 customize clients or resources and modify the behavior of method calls.
 
-Here is the list of events that users of Boto3 can register handlers to:
-
-* ``'creating-client-class'``
-* ``'creating-resource-class'``
-* ``'provide-client-params'``
+Here is a table of events that users of Boto3 can register handlers to. More information
+about each event can be found in the corresponding sections below:
+
+.. note::
+
+  Events with a ``*`` in their order number are conditionally emitted while all others are always emitted.
+  An explanation of all 4 conditional events is provided below.
```

Comment:
It looks like it's actually just 3 conditional events not 4

Reply:
yup, you're right